custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -338,10 +350,11 @@ def validate(self, dataframe):
 
         passes = conditions_map[self.condition](n_columns, test_value)
         if not passes:
-            return [
+            errors.append(
```

Comment:
can't we just get rid of the `errors` value and do the return statements?  i don't see the value in appending.